### PR TITLE
Update docker image name

### PR DESCRIPTION
**Pull request by jharrington22**
The PR updates the makefile to take an argument containing a docker image name

### CODE REVIEW

**Comment by jewzaam** [comment]
@jharrington22 why does this need to be done, there's a var already.  Is this an app-sre requirement for the name of the var?

**Comment by wshearn** [comment]
> @jharrington22 why does this need to be done, there's a var already. Is this an app-sre requirement for the name of the var?

yeah, the jenkins build script uses IMG

File: Makefile [hunk 1]
```diff
@@ -33,6 +33,9 @@ VERSION_FULL=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT
 
 OPERATOR_IMAGE_URI=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(VERSION_FULL)
```
Comment (jewzaam) [inline Makefile]:
```suggestion
IMG?=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(VERSION_FULL)
OPERATOR_IMAGE_URI=$(IMG)
```

**Comment by jharrington22** [comment]
@jewzaam we need to discuss the requirements of the app-sre pipeline vs how we want to build these operators. This update will allow the pipeline to deploy the operator without changing the operator or pipeline.

File: Makefile [hunk 2]
```diff
@@ -33,6 +33,9 @@ VERSION_FULL=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT
...
 
+# Image URL to use all building/pushing image targets
+IMG ?= ${OPERATOR_IMAGE_URI}
```
Comment (jewzaam) [inline Makefile]:
```suggestion

```

File: Makefile [hunk 3]
```diff
@@ -56,14 +59,14 @@ isclean:
 
 .PHONY: build
 build: isclean envtest
-	docker build . -f build/Dockerfile -t $(OPERATOR_IMAGE_URI)
+	docker build . -f build/Dockerfile -t ${IMG}
```
Comment (jewzaam) [inline Makefile]:
```suggestion
	docker build . -f build/Dockerfile -t ${OPERATOR_IMAGE_URI}
```

File: Makefile [hunk 4]
```diff
@@ -56,14 +59,14 @@ isclean:
...
 
 .PHONY: docker-build
 docker-build: build
 
 .PHONY: push
 push: build
-	docker push $(OPERATOR_IMAGE_URI)
+       docker push ${IMG}
```
Comment (jewzaam) [inline Makefile]:
```suggestion
       docker push ${OPERATOR_IMAGE_URI}
```

**Comment by jharrington22** [comment]
@jewzaam no problem, my miss understanding of the osd-operators-registry. Updated with your changes.

**Pull request merged**